### PR TITLE
style updates

### LINKED ISSUE #151: Pre-launch tidy ups
Misc. tidy ups to have consistent UI throughout app & comment out activity log placeholder since unused.

### PR DESCRIPTION
**Pull request by claireolmstead**
closes #151 

## Changes
- Start using 350px everywhere instead of some 350 and some 320.
- Add divider into transaction status component and remove from parent components.
- comment out activity log content.

### CODE REVIEW

File: test/unit-and-integration/provider.test.ts [hunk 1]
```diff
@@ -75,7 +74,6 @@ vi.mock('@polkadot/api', async () => {
 
 describe('Provider.svelte', () => {
   beforeEach(() => {
-    storeCurrentAction.set(ActionForms.NoForm);
```
Comment (shannonwells) [inline test/unit-and-integration/provider.test.ts:78]: is this unused now?

Comment (claireolmstead) [reply test/unit-and-integration/provider.test.ts:78]: Yep

File: src/components/ActivityLogPreview.svelte [hunk 2]
```diff
@@ -2,7 +2,8 @@
 </script>
 
 <!-- TODO: Connect activities -->
-<div class="min-w-[300px] border-l border-divider pl-5">
+<div></div>
+<!-- <div class="min-w-[300px] border-l border-divider pl-5">
```
Comment (shannonwells) [inline src/components/ActivityLogPreview.svelte:6]: can this be removed rather than commented out?

**Review by shannonwells — APPROVED** [review]
Looks good! I just had the one non-blocking question

File: src/components/ActivityLogPreview.svelte (hunk 2 above)
Comment (claireolmstead) [reply src/components/ActivityLogPreview.svelte:6]: In the next story, we're using this code, so keeping it in.

**Pull request merged**